packages/android_intent_plus/example/lib/main.dart

```diff
@@ -144,12 +149,19 @@ class ExplicitIntentsWidget extends StatelessWidget {
     intent.launch();
   }
 
-  void _openLinkInGoogleChrome() {
+  bool _intentResolved;
+
+  Future<void> _openLinkInGoogleChrome() async {
     final intent = AndroidIntent(
         action: 'action_view',
         data: Uri.encodeFull('https://flutter.dev'),
         package: 'com.android.chrome');
-    intent.launch();
+    bool intentResolved;
+    final bool result = await intent.canResolveActivity();
+    intentResolved = result;
+    setState(() {
+      _intentResolved = intentResolved;
+    });
```

Comment:
I'd rather not change the example

Reply:
I added this to show the result of canResolveActivity on UI with the help of Text, otherwise, it just fails and shows the error in the console. But if you want me to change it as it was before I will do that.